### DISCUSSION TITLE
notifyTesters cause upload to fail

### DISCUSSION THREAD

**Discussion by xiaohengpan** [wzieba/AppCenter-Github-Action#36] [General]
This seems really strange, but somehow when I had ` notifyTesters: false`, my script works properly. But when it's set to true, my android apk upload would fail.  

Just leaving this here as FYI.

**Comment by wzieba** [comment]
Hi @xiaohengpan, thanks for heads up! I've just run the test workflow and it seems that the upload worked with `notifyTesters: true` [see here](https://github.com/wzieba/AppCenter-Github-Action/runs/2350616405?check_suite_focus=true#step:7:1)

Could you please provide more information about failing build with `notifyTesters: true`? What were the other fields? Have you tried running action with `debug: true` and see for any issues there? Thanks!

**Reply by xiaohengpan** [reply]
Hi @wzieba , this is what my configuration looked like with a successful upload.
```    
- name: Upload artifact to AppCenter
      uses: wzieba/AppCenter-Github-Action@v1 
      with:
        appName: xiaohengpan/eProtect
        token: ${{secrets.APP_CENTER_TOKEN}}
        group: Collaborators
        file: examples/eprotect_demo/app/build/outputs/apk/debug/app-debug.apk
        gitReleaseNotes: true
        notifyTesters: false
        debug: true
        ```
        
With `notifyTesters` set to `true`, I was experiencing inconsistent results, ranging from failure to what it appears to be success, but with nothing uploaded to the app center. Here are couple screenshots ... 

1. Pipeline appears successful but nothing got uploaded to the Appcenter.
![Screen Shot 2021-04-16 at 12 02 00 PM](https://user-images.githubusercontent.com/74780418/115052561-13e0df80-9eac-11eb-96cc-8c5f77727b45.png)
2. Upload action failed.
![Screen Shot 2021-04-16 at 12 03 27 PM](https://user-images.githubusercontent.com/74780418/115052587-1d6a4780-9eac-11eb-9759-78e200216cf2.png)
